extensions/wikia/FounderEmails/events/FounderEmailsDaysPassedEvent.class.php

```diff
@@ -79,13 +79,13 @@ private function getEmailController( $activateDay ) {
 	}
 
 	public static function register( $wikiParams, $debugMode = false ) {
-		global $wgFounderEmailsEvents;
+		global $wgFounderEmailsEvents, $wgCityId;
 
 		$founderEmailObj = FounderEmails::getInstance();
 
 		$wikiFounder = $founderEmailObj->getWikiFounder();
-		$wikiFounder->setLocalPreference( "founderemails-joins", true );
-		$wikiFounder->setLocalPreference( "founderemails-edits", true );
+		$wikiFounder->setLocalPreference( "founderemails-joins", true, $wgCityId );
+		$wikiFounder->setLocalPreference( "founderemails-edits", true, $wgCityId );
```

Comment:
Isn't this case handled [here](https://github.com/Wikia/app/blob/SERVICES-783/includes/User.php#L2763) as long as `getLocalPreference` sets `$cityId` to `null` by default below?